### PR TITLE
Feature/#193

### LINKED ISSUE #193: Dark mode
- [ ] remove from toolbar control center (right icon in bottom bar) (iOS 13+)
- [ ] remove from settings
- [ ] connect to system settings (respond to OS notification)

### PR DESCRIPTION
**Pull request by shahenAnt**
## Ticket
fix #193 

## Description
Removed Display option from settings.
Removed 'Enable Night Mode' option from Toolbar control center If iOS version is 13.
Made connection between app interface style with system style only for iOS 13.

## Checklist:

- [ ] My squashed commit messages follows the [7 golden rules](https://chris.beams.io/posts/git-commit/)
- [ ] My code follows the code style of this project.
- [ ] I updated or write all necessary unit tests
- [ ] I updated all necessary documentation

### CODE REVIEW

File: Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -610,18 +610,6 @@ extension Strings {
...
-    public static let DisplayThemeAutomaticSwitchTitle = NSLocalizedString("Settings.DisplayTheme.SwitchTitle", comment: "Display (theme) settings switch to choose whether to set the dark mode manually, or automatically based on the brightness slider.")
-    public static let DisplayThemeAutomaticStatusLabel = NSLocalizedString("Settings.DisplayTheme.SwitchTitle", comment: "Display (theme) settings label to show if automatically switch theme is enabled.")
-    public static let DisplayThemeAutomaticSwitchSubtitle = NSLocalizedString("Settings.DisplayTheme.SwitchSubtitle", comment: "Display (theme) settings switch subtitle, explaining the title 'Automatically'.")
-    public static let DisplayThemeOptionLight = NSLocalizedString("Settings.DisplayTheme.OptionLight", comment: "Option choice in display theme settings for light theme")
-    public static let DisplayThemeOptionDark = NSLocalizedString("Settings.DisplayTheme.OptionDark", comment: "Option choice in display theme settings for dark theme")
```
Comment (chrmod) [inline Client/Frontend/Strings.swift]: please remove related Translations from string files.

File: UserAgent.xcodeproj/xcshareddata/xcschemes/Cliqz.xcscheme [hunk 2]
```diff
@@ -161,6 +161,15 @@
                <Test
                   Identifier = "DesktopModeTestsIphone">
                </Test>
+               <Test
```
Comment (chrmod) [inline UserAgent.xcodeproj/xcshareddata/xcschemes/Cliqz.xcscheme]: How about we remove those tests completely? 

On the other hand would nice to have at least a basic coverage for Dark Mode. Can we adopt any those tests to test iOS 13 notification?

Comment (winsmith) [reply UserAgent.xcodeproj/xcshareddata/xcschemes/Cliqz.xcscheme]: I'm all for removing the tests

Comment (shahenAnt) [reply UserAgent.xcodeproj/xcshareddata/xcschemes/Cliqz.xcscheme]: The dark mode on/off state we are getting when app is coming active. There is no iOS native notification for dark mode. So I need to investigate test mechanist to understand does it allow us to change system settings from test environment, etc.. @chrmod Do we need to do this investigation?

**Pull request merged**